catalog-rest-service/src/main/resources/json/schema/api/data/createTask.json

```diff
@@ -0,0 +1,65 @@
+{
+  "$id": "https://open-metadata.org/schema/api/data/createTask.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Create Task entity request",
+  "description": "Create Task entity request",
+  "type": "object",
+  "properties" : {
+    "name": {
```

Comment:
Do we need fully qualified name for task service.pipeline.task or tasks are independent?